src/services/story.ts

```diff
@@ -62,6 +62,20 @@ export default class StoryService {
             };
         }
         if (story.communityId !== undefined) {
+            const community = await this.community.findOne({
+                where: {
```

Comment:
No need for the entire table.
```suggestion
            const community = await this.community.findOne({
                attributes: ['id'],
                where: {
```